tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -901,6 +901,21 @@ Tensor xlogy(const Tensor& input_a, const Tensor& input_b, const MemoryConfig& o
     return operation::decorate_as_composite(__func__, _xlogy)(input_a, input_b, output_mem_config);
 }
 
+// Celu
+// torch.where(x > 0, x, alpha * (torch.exp(x / alpha) - 1))
+Tensor _celu(const Tensor& input_a, float alpha, const MemoryConfig& output_mem_config) {
+    float recip_val = 1.0f / alpha;
+    Tensor tmp = sub_unary(exp(mul_unary(input_a, recip_val, output_mem_config), output_mem_config), 1.0f, output_mem_config);
+    Tensor result = mul_unary(tmp, alpha, output_mem_config);
+    tmp.deallocate();
+    result = where(gtz(input_a, output_mem_config), input_a, result, output_mem_config);
```

Comment:
Can you use unary_chain for all these unary ops that are run back to back

Reply:
Hi @tt-aho ,
I guess the unary_chain is not possible here, since the tmp variable is storing in an intermediate output